doc/_static/references.bib

```diff
@@ -8,7 +8,7 @@
 
 @Article{Lee2001,
 author="Lee, Sang-Il",
-title="Developing a bivariate spatial association measure:  An integration of {Pearson's r and Moran's I}",
+title="Developing a bivariate spatial association measure:  An integration of {Pearson}'s r and {Moran's I}",
```

Comment:
Should this be {Pearson's r}?

Reply:
![image](https://user-images.githubusercontent.com/7359284/61964669-471a6700-af83-11e9-90b7-5b0e93420325.png)
https://link.springer.com/article/10.1007/s101090100064